andela/ah-titans-frontend
# #160244936 Implement designs for creating an article

#### What does this PR do?
Implement designs for creating an article.

#### Description of tasks to be completed
Implement designs for creating a new article and it should match the mockup design.

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ git checkout -b ft-implement-designs-for-creating-an-article-160244936
```
```sh
$ cd ah-titans-frontend
```

> Install the dependencies with the command below.

```sh
$ npm install
```
>Then run the app with the command below.
```sh
$ npm start
```
Now navigate to the URL "http://localhost:8080/article" and should see a page as the one shown on the screenshot below.

<img width="1280" alt="screen shot 2018-09-04 at 13 24 45" src="https://user-images.githubusercontent.com/27728464/45026104-f685dc80-b045-11e8-99ba-ffccb968c562.png">

#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160244936](https://www.pivotaltracker.com/story/show/160244936)
